vars/provisionNodes.groovy

```diff
@@ -163,239 +126,180 @@ def call(Map config = [:]) {
-                          fi'''
-    } else if (distro_type == "suse") {
-      if (repository_g != '') {
-        provision_script += '\nzypper --non-interactive ar -f ' +
-                            repository_g + ' daos-stack-group-repo'
-        provision_script += '\nzypper --non-interactive mr ' +
-                            '--gpgcheck-allow-unsigned-repo ' +
-                            'daos-stack-group-repo'
-      }
-      if (repository_l != '') {
-        provision_script += '\nzypper --non-interactive ar' +
-                            ' --gpgcheck-allow-unsigned -f ' +
-                            repository_l + ' daos-stack-local-repo'
-        provision_script += '\nzypper --non-interactive mr' +
+    if (inst_repos) {
+      provision_script += '\n' + iterate_repos +
+                          '''\nyum-config-manager --add-repo=''' + env.JENKINS_URL + '''job/daos-stack/job/\\\${repo}/job/\\\${branch//\\//%252F}/\\\${build_number}/artifact/artifacts/centos7/
+                             pname=\\\$(ls /etc/yum.repos.d/*.hpdd.intel.com_job_daos-stack_job_\\\${repo}_job_\\\${branch//\\//%252F}_\\\${build_number}_artifact_artifacts_centos7_.repo)
+                             if [ "\\\$pname" != "\\\${pname//%252F/_}" ]; then
+                                 mv "\\\$pname" "\\\${pname//%252F/_}"
+                             fi
+                             pname="\\\${pname//%252F/_}"
+                             sed -i -e '/^\\[/s/%252F/_/g' -e '\\\$s/^\\\$/gpgcheck = False/' "\\\$pname"
+                             cat "\\\$pname"
+                         done'''
+    }
+    if (inst_rpms) {
+      provision_script += '''\nyum -y erase ''' + inst_rpms
+    }
+    gpg_key_urls.each { gpg_url ->
+      provision_script += '\nrpm --import ' + gpg_url
+    }
+    provision_script += '''\nrm -f /etc/profile.d/openmpi.sh
+                             rm -f /tmp/daos_control.log
+                             yum -y erase metabench mdtest simul IOR compat-openmpi16
+                             yum -y install epel-release
+                             if ! yum -y install CUnit fuse python36-PyYAML   \
+                                                 python36-nose                \
+                                                 python36-pip valgrind        \
+                                                 python36-paramiko            \
+                                                 python2-avocado              \
+                                                 python2-avocado-plugins-output-html \
+                                                 python2-avocado-plugins-varianter-yaml-to-mux \
+                                                 libcmocka python-pathlib     \
+                                                 python2-numpy git            \
+                                                 python2-clustershell         \
+                                                 golang-bin ipmctl ndctl'''
```

Comment:
I'd like to see another flavour of CI image one day that includes all of this static stuff that we take time to install on every test run.

Reply:
> All this stuff is being baked into the new CI packer image.

Excellent.  So can/should it be removed from here?

> Eventually we want to be using the information in the RPM or other package to make sure that the dependencies are present instead of having to code them in here

I'm not sure all of those dependencies will ever be in one or more RPMs.

Reply:
This needs to be present until daos pr-2008 is landed and all existing existing PRs updated to use that commit.

We should have some type of file in the daos repository and or some sort of tool that will report to us all the dependent packages needed for testing DAOS.